index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+     <div class="container">
+        <!-- header -->
+        <header>
+            <nav>           
+                 <h1>Alishah</h1>
+                 <a href="#"><img src="Image/Union.png" alt="button"></a>
+            </nav>
```

Comment:
- [ ] I think it would be better for the logo  **(Alishah)** to use an ```<a>``` tag. This is for three reasons: **convention, semantics,** and **user experience(UX)**. [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 😎😎
- [ ] It would be better to use the ```nav``` tag for the hamburger button and navigation links only.
Remember, the <nav> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a section of a page whose purpose is to provide navigation links. 🙏 

example:
```
    <a>Alishah</a>
        <nav>           
         
             <a href="#"><img src="Image/Union.png" alt="button"></a>
        </nav>